dev-util/clinfo/clinfo-2.1.17.02.09.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Generic tool to print information about all available OpenCL platforms and devices"
```

Comment:
Didn't repoman complain about long description? Try to cut it a little bit shorter.